Error during compilation of tests

The tests can not be compiled with OpenSSL 1.1.0 on Windows:

```
 d:\projects\opcuastack\tst\opcuastackcore\certificate\rsasign_t.cpp(27): error C2027: use of undefined type "evp_pkey_st"
```
On Linux the erorr is the same.

--- a/CHANGELOG.rst
+++ b/CHANGELOG.rst
@@ -9,6 +9,7 @@ Release 3.6.0
 **Bugs**:
  
 * fix too specific params of *build.bat* #16, by @flipback
+* fix OpenSSL-1.1 compatibility in test compilation #46, by @flipback
 
 
 Release 3.5.1 (2018-11-11)

--- a/tst/OpcUaStackCore/Certificate/RSASign_t.cpp
+++ b/tst/OpcUaStackCore/Certificate/RSASign_t.cpp
@@ -2,6 +2,7 @@
 
 #include "OpcUaStackCore/Certificate/RSAKey.h"
 #include "OpcUaStackCore/Certificate/PrivateKey.h"
+#include "OpcUaStackCore/Certificate/OpenSSLCompat.h"
 #include "unittest.h"
 #include <openssl/bio.h>
 #include <openssl/rsa.h>
@@ -24,7 +25,7 @@ BOOST_AUTO_TEST_CASE(RSASign_sign)
 	RSAKey rsaKey(2048);
 	PrivateKey privateKey = rsaKey.privateKey();
 	EVP_PKEY* key = privateKey;
-	RSA* r = key->pkey.rsa;
+	RSA* r = EVP_PKEY_get0_RSA(key);
 
 	unsigned char *sig = NULL;
 	unsigned int sig_len = 0;

